### DISCUSSION TITLE
DRS markings on track map

### DISCUSSION THREAD

**Discussion by BeyondVertical** [f1multiviewer/issue-tracker#236] [Feature requests] [answered]
It would be awesome to see the DRS zones on the track map and maybe grey them out (or something like that) when they are disabled.

**Comment by Larsen127** [comment]
DRS detection zone would be great too. Probably even more useful imo

**Reply by BeyondVertical** [reply]
Would be a great addition too, yes.

**Comment by Broukmiken** [comment]
yes, i wish we could have this addition too
![Capture d'écran 2024-08-01 230601](https://github.com/user-attachments/assets/26321ea4-df9a-40a0-a00b-e7c3a469cbb4)


![mod](https://github.com/user-attachments/assets/3122d5ff-f39e-4ccd-a2bb-487329bc1b52)

**Comment by tango2590** [comment]
Not only DRS zones, but sectors too. I feel like those are the only things missing with the track map!

**Comment by jeppeolesen** [comment, marked as answer]
This has been implemented and released in MultiViewer 2.0, which is now available in the beta release channel.

In order to use the DRS zone features, you need to be a sponsor.

![image](https://github.com/user-attachments/assets/3afce026-02f6-4c79-831f-8890b1328562)
